FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -1787,6 +2020,71 @@ components:
         - sourceId
         - description
         - timeOfBlocklist
+           
+    UnavailabilitySchedule:
+      type: object
+      required:
+        - componentId
+        - schedules
+      properties:
+        componentId:
+          type: string
+          format: uuid
+          description: |
+            The unique identifier for the component. 
+            - This can represent one or more component IDs when the schedule is applied to multiple components.
```

Comment:
This description is incorrect - we are in a single UnavailabilitySchedule object, and there is not a way to specify multiple components here (unless you change this data type to be an array of UUIDs)

Reply:
You are right, I originally meant multiple UnavailabilitySchedule objects in a single request, each with its own componentId. Good catch. 

fixed description